lib/signaling/v2/room.js

```diff
@@ -340,7 +344,12 @@ class RoomV2 extends RoomSignaling {
         // NOTE(mroberts): _teardownDominantSpeakerSignaling was called.
         return;
       }
-      const dominantSpeakerSignaling = new DominantSpeakerSignaling(receiver.toDataTransport());
+
+      // NOTE(mpatwardhan): The underlying RTCDataChannel is closed whenever
+      // the VMS instance fails over, and a new RTCDataChannel is created in order
+      // to resume sending Dominant Speaker updates.
+      receiver.on('close', () => this._teardownDominantSpeakerSignaling());
```

Comment:
@makarandp0 "receiver.**once**(" ? Also can you add an extra line after this line. Usually its good to separate the code which has comments for it from the rest of the code.